lib/body.js

```diff
@@ -15,37 +15,37 @@ const webStreams = require('node-web-streams');
  */
 class Body {
     constructor(body, opts) {
-		opts = opts || {};
+        opts = opts || {};
```

Comment:
[:+1:](https://medium.com/@hoffa/400-000-github-repositories-1-billion-files-14-terabytes-of-code-spaces-or-tabs-7cfe0b5dd7fd)  <-- click